R/loop_over_all_stages-helpers.R

```diff
@@ -561,6 +561,13 @@ attach_pkgs <- function(pkgs, load = character(0)) {
 
   # There may be some packages that need to be fully loaded to work
   # appropriately.
+
+  # These are packages that users might directly interact with so we should
+  # fully load them
+  load_user_pkgs <- c("parsnip", "recipes", "workflows", "tailor", "tune")
```

Comment:
I don't know if this is enough or if we should just fully load everything (which I think is what happened previously)